process-document/src/main/kotlin/com/ritense/processdocument/listener/ProcessDocumentLinkEventListener.kt

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2015-2025 Ritense BV, the Netherlands.
+ *
+ * Licensed under EUPL, Version 1.2 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ritense.processdocument.listener
+
+import com.ritense.authorization.annotation.RunWithoutAuthorization
+import com.ritense.processdocument.service.CaseDefinitionProcessLinkService
+import com.ritense.valtimo.contract.annotation.SkipComponentScan
+import com.ritense.valtimo.contract.event.CaseDefinitionCreatedEvent
+import io.github.oshai.kotlinlogging.KotlinLogging
+import org.springframework.context.event.EventListener
+import org.springframework.stereotype.Component
+import org.springframework.transaction.annotation.Transactional
+
+@Transactional
+@Component
+@SkipComponentScan
+class ProcessDocumentLinkEventListener(
+    private val caseDefinitionProcessLinkService: CaseDefinitionProcessLinkService
+) {
+
+    @RunWithoutAuthorization
+    @EventListener(CaseDefinitionCreatedEvent::class)
+    fun handleCaseDefinitionCreatedEvent(event: CaseDefinitionCreatedEvent) {
```

Comment:
This listener is also needed here `@EventListener(CaseDefinitionPreDeleteEvent::class)`
Because when a draft is deleted, the process document link should also be deleted.

Reply:
Fixed, thanks for pointing this out!